src/components/Navbar/Navbar.js

```diff
@@ -1,54 +1,57 @@
 import React from 'react';
 import { Nav, Navbar } from 'react-bootstrap';
 import { LinkContainer } from 'react-router-bootstrap';
+import { useLocation } from 'react-router-dom';
 import logo from '../../assets/logo.png';
 import './Navbar.css';
 
-const NavBar = () => (
-	<Navbar className='main-nav' fluid='true' collapseOnSelect expand='lg'>
-		<LinkContainer to='/'>
-			<Navbar.Brand>
-				<img src={logo} className='App-logo' alt='logo' />
-			</Navbar.Brand>
+function ActiveListItem(props) {
+	// active={false} prevents a link from staying active when on the main page
+	return (
+		<LinkContainer to={props.url}>
+			<Nav.Link active={false}>{props.name}</Nav.Link>
 		</LinkContainer>
-		<Navbar.Toggle aria-controls='responsive-navbar-nav'>
-			{' '}
-			<i className='fas fa-bars fa-lg'></i>
-		</Navbar.Toggle>
-		<Navbar.Collapse className='justify-content-end' id='responsive-navbar-nav'>
-			<Nav>
-				<LinkContainer to='/about'>
-					<Nav.Link>About</Nav.Link>
-				</LinkContainer>
-				<LinkContainer to='/calendar'>
-					<Nav.Link>Events</Nav.Link>
-				</LinkContainer>
-				<LinkContainer to='/gallery'>
-					<Nav.Link>Gallery</Nav.Link>
-				</LinkContainer>
-				<LinkContainer to='/tutoring'>
-					<Nav.Link>Tutoring</Nav.Link>
-				</LinkContainer>
-				<LinkContainer to='/membership'>
-					<Nav.Link>Membership</Nav.Link>
-				</LinkContainer>
-				{/* <LinkContainer to='/hackathons'>
-					<Nav.Link>Hackathons</Nav.Link>
-				</LinkContainer> */}
-				<LinkContainer to='/contactus'>
-					<Nav.Link>Contact Us</Nav.Link>
-				</LinkContainer>
-				<Nav.Link href='https://uhcode.red/' target='_blank' rel='noreferrer'>
-					CodeRED <i className='fas fa-external-link-alt'></i>
-				</Nav.Link>
-				<LinkContainer to='/register'>
-					<Nav.Link>
-						<button className='button'>Register</button>
+	);
+}
+
+const NavBar = () => {
+	const { pathName } = useLocation();
+	console.log(pathName);
```

Comment:
**suggestion:** we don't need to print the pathName on live sites

Reply:
Very true did not mean to leave that in lmao Woops